examples/graphql-server-typescript/src/index.ts

```diff
@@ -21,6 +21,7 @@ const start = async () => {
   const accountsDb = new DatabaseManager({
     sessionStorage: userStorage,
     userStorage,
+    mfaLoginAttemptsStorage: userStorage,
```

Comment:
Make sure `mfaLoginAttemptsStorage` is optional so that it would not break the current usages. Also no need for it when MFA is not configured